mantidproject/mantidimaging
# Removes pyqt and pyqt5 from meta.yaml for conda builds

This removes an unnecessary Qt installation. The necessary Qt library files are installed from `pip install pyqt5`

To test:
- Builds should pass
- Check that package builds locally.
- I'm not sure that you can install the local package & get dependency resolution - it seems `conda install` doesn't resolve deps for local files

Fixes #731
